credential/zkpass/v0.1.0/schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/credential/zkpass/v0.1.0/schema.json",
+    "title": "Schema for ZKPass",
+    "description": "A schema for zkpass credential",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "zkpass_credentials",
+        "indexes": {
+            "taskId": ["taskId"]
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "https://common.schemas.verida.io/core/base/schema.json"
```

Comment:
URL doesn't exist.

Should be `https://core.schemas.verida.io/base/latest/schema.json`

Reply:
Done